src/main/webapp/i18n/de/global.json

```diff
@@ -300,7 +300,8 @@
             "number": "Dieses Feld muss eine Zahl sein.",
             "datetimelocal": "Dieses Feld muss eine Datums- und Zeitangabe enthalten.",
             "patternLogin": "Dieses Feld darf nur Buchstaben, Ziffern und E-Mail-Adressen enthalten.",
-            "invalidPattern": "Dieses Feld beinhaltet ein ungültiges E-Mail Muster."
+            "invalidPattern": "Dieses Feld beinhaltet ein ungültiges E-Mail Muster.",
+            "invalidDate": "Dieses Feld beinhaltet ein ungültiges Datum. Das Datum soll MM TT JJ oder ähnliches Format haben."
```

Comment:
I think this might depend on the browser. On chrome many different formats worked, `MM TT JJ` included. This is why I wrote "or a similar format". I now changed the it, so it says to use the `MM/DD/YYYY HH:mm` format, since it seems to work in also seems to work in other browser.